pygeo/mphys/mphys_dvgeo.py

```diff
@@ -6,39 +6,52 @@
 except ImportError:
     # not everyone might have openvsp installed, and thats okay
     pass
+
+try:
+    from .. import DVGeometryESP
+except ImportError:
+    # not everyone might have esp installed, and thats okay
+    pass
+
 from mpi4py import MPI
 import numpy as np
 
 # class that actually calls the dvgeometry methods
 class OM_DVGEOCOMP(om.ExplicitComponent):
     def initialize(self):
 
-        self.options.declare("ffd_file", default=None)
-        self.options.declare("vsp_file", default=None)
-        self.options.declare("vsp_options", default=None)
+        self.options.declare("geo_file", default=None)
+        self.options.declare("geo_type", default=None)
```

Comment:
I see later in the code there are still references to `self.geo_type`. Is this declared option here automatically converted to a member variable or do we need to do that ourselves and set `self.geo_type = self.options["geo_type"]`?

Reply:
I'd say saving it as self.geo_type is fine because otherwise you go through one attribute and one dictionary.

Reply:
the option is not converted to attribute automatically, we have to save it. sorry I didnt read the question fully. we can save it when we read the input parameters.

Reply:
Sounds good! I think the code as it is is broken since we don't save the option saved. Once that is saved it should be fixed.